framework-tests/app/src/androidTest/java/com/samsungxr/stress/StressTests.java

```diff
@@ -140,18 +141,20 @@ public void gcOomTest2() throws Exception {
     private void oomTest(boolean createBitmap) throws IOException {
         try {
             final int MaxInstances = 100000;
-            SXRContext sxrContext = ActivityRule.getActivity() .getSXRContext();
+            SXRContext sxrContext = ActivityRule.getActivity().getSXRContext();
+            SXRTexture texture = new SXRTexture(sxrContext);
+
             for (int count = 0; count < MaxInstances; count++) {
                 Log.d(TAG, "Count: " + count);
                 SXRNode sceneObject = new SXRNode(sxrContext, sxrContext.createQuad(10f, 10f));
                 SXRRenderData renderData = sceneObject.getRenderData();
-                SXRTexture texture;
+                texture.clearData();
```

Comment:
I find this clearData confusing. Immediately a new instance will be stored in "texture". If all is good, the old texture will be destroyed and dtor calls clearData. If there is really a need to call clearData explicitly here, it is a sign that something is not working as intended.